includes/services.inc.php

```diff
@@ -129,7 +129,7 @@ function poll_service($service)
 
     // If we do not have a cmd from the check script, build one.
     if ($check_cmd == "") {
-        $check_cmd = $config['nagios_plugins'] . "/check_" . $service['service_type'] . " -H " . ($service['service_ip'] ? $service['service_ip'] : $service['hostname']);
```

Comment:
Thanks for explaining! 

> If a service check doesn't need the -H then you create a custom service check file like we already have, see includes/services/check_procs.inc.php

Where I should create custom service check file? How to use it?